packaging/deb-systemd/debian/mackerel-agent.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=mackerel.io agent
+Documentation=https://mackerel.io/
+After=network-online.target nss-lookup.target
+
+[Service]
+Environment=MACKEREL_PLUGIN_WORKDIR=/var/tmp/mackerel-agent
+Environment=ROOT=/var/lib/mackerel-agent
+EnvironmentFile=-/etc/default/mackerel-agent
+ExecStartPre=/bin/mkdir -m 777 -p $MACKEREL_PLUGIN_WORKDIR
+ExecStart=/usr/bin/mackerel-agent supervise --root $ROOT $OTHER_OPTS
+ExecStopPost=/bin/sh -c '[ "$AUTO_RETIREMENT" = "" ] || [ "$AUTO_RETIREMENT" = "0" ] && true || /usr/bin/mackerel-agent retire -force --root $ROOT $OTHER_OPTS'
```

Comment:
Originally `==` is used here, but it's bash extented syntax.
In Debian /bin/sh is symlinked to dash, so I need to fix at https://github.com/mackerelio/mackerel-agent/pull/360/commits/a811463951eb1e8066ca37b9d4e7d14204d9cbdc

BTW should I also change `ExecStopPost` in CentOS systemd service too?